doc/user/bgp.rst

```diff
@@ -636,6 +635,19 @@ Defining Peer
 
       can't find neighbor 10.0.0.1
 
+.. index:: neighbor PEER remote-as internal
+.. clicmd:: neighbor PEER remote-as internal
+
+   Create a peer as you would when you specify an ASN, except that if the
+   peers ASN is different than mine as specified under the `router bgp ASN`
+   command the connection will be denied.
+
+.. index:: neighbor PEER remote-as external
+.. clicmd:: neighbor PEER remote-as external
+
+   Create a peer as you would when you specify an ASN, except that if the
+   peers ASN is the same as mine as specified under the `router bgp ASN`
```

Comment:
If you change this to
```
:clicmd:`router bgp ASN`
```
it will automatically link to the command definition. Otherwise, the syntax for inline code in RST is

    ``double backticks``

Reply:
changed